packages/x-date-pickers/src/AdapterLuxon/AdapterLuxon.ts

```diff
@@ -480,9 +470,7 @@ export class AdapterLuxon implements MuiPickersAdapter<DateTime, string> {
   };
 
   public getWeekNumber = (value: DateTime) => {
-    // TODO: remove when `@types/luxon` add support for the parameter.
-    // @ts-ignore
-    return value.localeWeekNumber ?? value.weekNumber;
+    return value.localWeekNumber ?? value.weekNumber;
```

Comment:
Looks like there was a bug here.
`localeWeekNumber` does not seem to exist, only `localWeekNumber`.

Reply:
Oppsie !